src/main/java/uk/gov/hmcts/cft/idam/testingsupportapi/config/SchedulerConfig.java

```diff
@@ -25,7 +25,7 @@ public void triggerExpiredBurnerUsersTask() {
 
     @Scheduled(initialDelayString = "${scheduler.initialDelayMs}",
         fixedRateString = "${scheduler.session.triggerExpiryFrequencyMs}")
-    public void triggerExpiredBurnerSessionsTask() {
+    public void triggerExpiredSessionsTask() {
```

Comment:
can we call these methods something super obvious like runExpiredSessionCleanup (if thats what it does)
I dont immediately know what a expired session task does

Reply:
Yeah, these are essentially just the bean names, so I'll rename them in another PR.